gslab_scons/_release_tools.py

```diff
@@ -10,29 +10,30 @@
 
 from _exception_classes import ReleaseError
 
-
-def release(vers, org, repo, 
+def release(vers, org, repo,
             DriveReleaseFiles = [],  
             local_release     = '',  
             target_commitish  = '', 
-            zip_release       = True):
+            zip_release       = True,
+            token             = None):
     '''Publish a release
 
     Parameters
     ----------
     env: an SCons environment object
     vers: the version of the release
     DriveReleaseFiles a optional list of files to be included in a
-        release to Google Drive.
+        release to drive (e.g. DropBox or Google Drive).
```

Comment:
The idea is that someone should be able to use the tool in `release.py` and specify what they want to release. I added an option to `release.py` to add your own file list, otherwise as default we crawl through `/release` and release everything in there.